server/cri/types/types.go

```diff
@@ -494,14 +494,16 @@ func NewLinuxContainerConfig() *LinuxContainerConfig {
 }
 
 type LinuxContainerResources struct {
-	CPUPeriod          int64
-	CPUQuota           int64
-	CPUShares          int64
-	MemoryLimitInBytes int64
-	OomScoreAdj        int64
-	CPUsetCPUs         string
-	CPUsetMems         string
-	HugepageLimits     []*HugepageLimit
+	CPUPeriod              int64
+	CPUQuota               int64
+	CPUShares              int64
+	MemoryLimitInBytes     int64
+	OomScoreAdj            int64
+	CPUsetCPUs             string
+	CPUsetMems             string
+	HugepageLimits         []*HugepageLimit
+	Unified                map[string]string
+	MemorySwapLimitInBytes int64
```

Comment:
Unless there's a reason not to, my OCD would appreciate ordering the variables in alphanumeric order.

Reply:
It makes `git annotate` harder to use when attempting to determine when a field or feature was added?